bluesky_kafka/tests/test_kafka.py

```diff
@@ -3,25 +3,23 @@
 import multiprocessing
 import pprint
 import time
-
 import numpy as np
 import pickle
```

Comment:
Looks like I did not order the imports according to PEP8.

Reply:
Updated